### ISSUE TITLE
Generated code with reserved word parameter does not compile

### ISSUE THREAD

**Issue by stuarthendren** [vojtechhabarta/typescript-generator#481]
If a rest method declares a reserved word as a path variable e.g. `@PathVariable("class")` then the generated code uses the reserved word as the parameter name e.g. `class` which does not compile.

Maybe this case could be handled by the generator or by a parameter name mapping configuration? Maybe there already is a way to handle this case?

Rough Example

```java
@GetMapping(value = "/logs/{class}")
public Collection<String> getLogs(@PathVariable("class") String clazz)
```

Gives something like the following which errors:

```ts
 getLogs_class(
    class: string
  ): RestResponse<string[]> {
    return this.httpClient.request({
      method: 'GET',
      url: uriEncoding`logs/${class}`,
    })
  }
```

**Comment by vojtechhabarta** [comment]
Thanks for reporting this problem (with example).
I committed fix which prefixes reserved words with `_` character. I don't think it is worth to add parameter mapping feature.

**Comment by stuarthendren** [comment]
Thank you for the fast response.

**Comment by vojtechhabarta** [comment]
Released in [2.24.612](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.24.612).
